ckb.json

```diff
@@ -0,0 +1,123 @@
+{
+   "ckb":{
```

Comment:
```suggestion
   "ku":{
```

Should this be `ku` i know it is in Java Locale and Browser: https://stackoverflow.com/a/73295301/502366

Reply:
Hello, no please don't change it to ku, because that's for Kurdish ([Kurmanji](https://en.wikipedia.org/wiki/Kurmanji)). [ckb](https://en.wikipedia.org/wiki/Kurdish_languages) is for Sorani Kurdish

Reply:
Understood was just checking!